src/main/java/org/jenkinsci/plugins/displayurlapi/DisplayURLProvider.java

```diff
@@ -28,8 +38,7 @@ public abstract class DisplayURLProvider implements ExtensionPoint {
      * @return DisplayURLProvider
      */
     public static DisplayURLProvider get() {
-        DisplayURLProvider preferredProvider = getPreferredProvider();
-        return preferredProvider != null ? preferredProvider : DisplayURLProviderImpl.INSTANCE;
+        return DisplayURLProviderImpl.INSTANCE;
```

Comment:
As before #42. See discussion in [JENKINS-69006](https://issues.jenkins.io/browse/JENKINS-69006).